include/openssl/x509_vfy.h

```diff
@@ -18,7 +18,6 @@
 # endif
 
 # include <openssl/opensslconf.h>
-# include <openssl/lhash.h>
```

Comment:
-1, public header.

Reply:
Just in case, same can be said about #4430.

Reply:
I've reverted the public header changes in #4430 in #4437